packages/contracts/src/dollar/ERC1155SetUri/ERC1155SetUri.sol

```diff
@@ -59,7 +66,13 @@ contract ERC1155SetUri is Context, ERC165, IERC1155, IERC1155MetadataURI {
      * Clients calling this function must replace the `\{id\}` substring with the
      * actual token type ID.
      */
-    function uri(uint256) public view virtual override returns (string memory) {
+    function uri(uint256)
+        public
+        view
+        virtual
+        override
+        returns (string memory)
+    {
```

Comment:
All of the formatting changes are unexpected. What formatter are you using here and how exactly are you invoking it?

Reply:
Prettier. I started getting formatting errors when i tried to make a commit. I ran the Prettier format command as specified in the package.json file.